themes/theme-docs/src/components/Card.styles.ts

```diff
@@ -2,7 +2,7 @@ import { ThemeComponent, cva } from '@marigold/system';
 
 export const Card: ThemeComponent<'Card'> = cva(
   [
-    'bg-bg-surface border-secondary-300 relative overflow-hidden rounded-xl border shadow',
+    'bg-bg-surface border-secondary-300 relative w-full overflow-hidden rounded-xl border shadow',
```

Comment:
should they really take full width? Looks a bit weird when only one item is there

Reply:
fixed